### PR TITLE
Newcomers_Guide: Forbid mentioning unrelated users

### PR DESCRIPTION
**Pull request by adtac**

### CODE REVIEW

File: docs/Developers/Newcomers_Guide.rst [hunk 1]
```diff
@@ -122,6 +122,11 @@ keep in mind.
     may be signed into, you might even wake someone on the other side of the
     world up. Also it discourages other people to answer the question,
     so you might wait longer for an answer.
+  * Do not mention users unrelated to a particular a issue/question. This also
```
Comment (Makman2) [inline docs/Developers/Newcomers_Guide.rst]: to a particular ~~a~~ issue/question

**Comment by Makman2** [comment]
ack 13dbd7a

**Comment by cornu-ammonis** [comment]
ack 13dbd7

**Comment by Makman2** [comment]
@gitmate-bot rebase

**Comment by adtac** [comment]
@rultor merge

**Pull request merged**